wemake_python_styleguide/visitors/ast/attributes.py

```diff
@@ -23,14 +23,136 @@ class WrongAttributeVisitor(BaseNodeVisitor):
         'mcs',
     ))
 
-    _allowed_magic_attributes: ClassVar[FrozenSet[str]] = frozenset((
-        '__class__',
-        '__name__',
-        '__qualname__',
-        '__doc__',
-        '__subclasses__',
-        '__mro__',
-        '__version__',
+    _disallowed_magic_attributes: ClassVar[FrozenSet[str]] = frozenset((
```

Comment:
Let's move this to `constants.py`, it is rather long!

It would also be a good idea to link to it from the violation's docs.

Thanks a lot for your time and effort 👍